### ISSUE TITLE
Upgrade install opens `Example.xml`

### ISSUE THREAD

**Issue by kdschlosser** [EventGhost/EventGhost#83]
After an upgrade of EG it opens the Example.xml. I don't think this is a bug but just hasn't been coded into EG but have the upgrade open the default config file add any of the persistent data in there from plugins to the new config file and also grab the save file location and name to run at startup.

**Comment by blackwind** [comment]
I'm unable to reproduce, but I'll leave this open for now just in case it's actually a bug.

**Comment by kdschlosser** [comment]
when i installed this beta it moved all of the information from the roaming/EventGhost folder to roaming/EventGhost.bak

and created a whole new config.py file

i had EG running when i launched the installer so that may have been the catalyst

**Comment by blackwind** [comment]
Because the v0.4.1 installers remove that folder on uninstall, the new installer renames it to `EventGhost.bak`, uninstalls your previous version of EventGhost, and then renames back to `EventGhost`. We have to uninstall the previous version so Python 2.6 stuff doesn't get mixed in with the new stuff.

The only reason I can think of as to why it would rename to `EventGhost.bak` but fail to revert that change is if you ran the installer, got the "EventGhost is running" warning, closed EventGhost, hit OK, started EventGhost back up again during the install for some reason, and ignored all the "this file is in use" errors.

If this isn't user error and you can come up with a way to reliably reproduce, I'll definitely remove the `invalid` tag here and begin work on a fix.

**Comment by kdschlosser** [comment]
I will see if I can do it. But what I think happened was I started the install EG was running. I have been working on a plugin so when I closed EG it hung and I didn't notice it. I hit the OK and it told me again that it was still running. Then I killed the EG process and then I hit the OK again . And the installer did its thing. I will try to come up with an exact procedure to reproduce this. But it could be such an obscure way it happened it's not worth the time to correct.
